README.md

```diff
@@ -1,34 +1,36 @@
 
-Run `./aue.sh` from the home directory. Results are found in the `output` directory. Each row of the output file contains the BBLs for a single solution. The number of columns is the total number of possible establishments. These counts are also output to the terminal.
+## Background
+
+### Usage
 
-## Overview of Results
+Run `./aue.sh` from the home directory. Results are found in the `output` directory. Each row of the output file contains the BBLs for a single solution. The number of columns is the total number of possible establishments. These counts are also output to the terminal.
 
-Using the methodology outlined below, we have identified a best-case scenario of **339** lots that could feasibly have proximity-restricted establishments, given a 500ft restriction. The worst-case scenario, meaning the case where establishments are arranged in a way that limits the addition of new establishments, is **333** lots. Note that these numbers are the maximum number of *possible* establishments. In reality, not all lots are currently vaccant or suitable for future shops.
+### Overview of Results
 
+Using the methodology outlined below, we have identified a best-case scenario of **339** lots that could feasibly have proximity-restricted establishments, given a 500ft restriction. The worst-case scenario, meaning the case where establishments are arranged in a way that limits the addition of new establishments, is **333** lots. Note that these numbers are the maximum number of *possible* establishments. In reality, not all lots are currently vacant or suitable for future shops.
 
-## Methodology
+### Methodology
 
-We approached this problem by creating a matrix of possible co-existing establishments -- lots that can simultaneaously have establishments because they are further than 500ft apart. Simplifying the distance calculations to a binary flag makes our calculations much easier. 
+We approached this problem by creating a matrix of possible co-existing establishments -- lots that can simultaneously have establishments because they are further than 500ft apart. Simplifying the distance calculations to a binary flag makes our calculations much easier.
 
-### Data loading and preparation
+#### Data loading and preparation
 
 **Input data**:
 
-The input data for this analysis is the subset of MapPLUTO lots that meet the criteria, buffered to 500ft. See the image below for an example of what this data looks like.
+The input data for this analysis is the subset of MapPLUTO lots that meet the exclusion criteria. See the image below for an example of what this data looks like.
```

Comment:
Maybe we can add that this data comes from GIS and that it has to be archived via postgres into `EDM-DATA`?

Reply:
added in https://github.com/NYCPlanning/aue/pull/11/commits/648f7d1b1dec4b938983f44e8c09859f6655256a